content/migration-from-otrs-6.rst

```diff
@@ -287,10 +287,15 @@ by the script *bin/backup.pl*.
 Alternatively, the database can be dumped on another server and be transferred to the Docker host afterwards.
 Here are sample commands that achieve this goal.
 
+.. warning::
+
+    With these commands any special setup of MySQL collations is removed.
+    In case special collations are needed, they have to be readded manually.
```

Comment:
In case you need any special collations, make sure to re-add them manually.

Reply:
Done